test_values/test_cases.json

```diff
@@ -468,30 +468,30 @@
       ]
     },
     {
-      "leaf_values": [0, 1, 2, 3, 4, 5, 6, 7],
-      "target_values":[0, 1, 2, 3, 4, 5, 6, 7],
-      "roots":[
```

Comment:
Seems odd to me that these changes would be here when the name changes happened at the previous commit.

Reply:
It's because of the struct fields, I may fix this if you think it's better.

Reply:
Could you explain by what you mean when you say it's because of the struct fields?

My reasoning for this was because all the `test_cases.json` changes seem to be in the previous commit but these were in the later commit.

Reply:
Hmm, I see. This field naming should change in the first commit. Now I think it's ok.